.github/workflows/docs.yml

```diff
@@ -15,21 +15,21 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
```

Comment:
Why commit hash instead of just `v4.1.1`?

Reply:
Using hash was originally introduced in some of other our repositories, it covers the extremely rare case that the tag has been re-used a later point in time.